app/figures.py

```diff
@@ -9,6 +12,54 @@
 time_range = ["2035-01-22 00:00:00", "2035-01-22 00:07:01.140"]
```

Comment:
Can you update this time range to go from "2035-01-22 04:00:00.000" to "2035-01-22 11:01:08.400"? I pushed directly to main when I updated the time step in the pre-set data.

```suggestion
time_range = ["2035-01-22 04:00:00", "2035-01-22 11:01:08.400"]
```

Reply:
I see you already have this change in the Wesim PR #89